install.sh

```diff
@@ -163,7 +163,8 @@ install_nvm_from_git() {
       }
     else
       # Cloning repo
-      command git clone "$(nvm_source)" --depth=1 "${INSTALL_DIR}" || {
+      command git clone "$(nvm_source)" --depth=1 -o origin "${INSTALL_DIR}" 2> /dev/null \
+        || command git clone "$(nvm_source)" --depth=1 "${INSTALL_DIR}" || {
```

Comment:
```suggestion
        || command git clone "$(nvm_source)" --depth=1 -o origin "${INSTALL_DIR}" || {
```

Reply:
I'll just shrink it to the single command/line since at least `-o origin` looks like it'll work with the version restriction.

Reply:
The difference is that the first one suppresses stderr; it's intentional to have the command twice so that when it fails, it tries again and prints out stderr for debugging purposes.